dev/db2/latest/docker-compose.yml

```
@@ -0,0 +1,20 @@
+services:
+  db2-latest:
+    container_name: sequelize-db2-latest
+    image: icr.io/db2_community/db2:11.5.8.0
```

```suggestion
    image: ibmcom/db2-amd64:11.5.8.0
```
I'd prefer to remain using the same image as before, unless you know that this source won't be updated

---

The only reason I changed it as I saw [this message](https://community.ibm.com/community/user/datamanagement/blogs/vijaya-katikireddy/2023/02/04/db2-community) which is on the [ibmcom/db2](https://hub.docker.com/r/ibmcom/db2) docker registry.

---

In that case; can you update the oldest as well?

---

There isn't a tag for 11.5.6.0a.
The available tags for icr.io/db2_community/db2 are:
- 11.5.8.0
- 11.5.7.0a
- 11.5.5.1

---

If tests pass on 11.5.5.1 we can move to that, as long as we update it to the CONTRIBUTING file and the version policy page on the website

---

I've just run the tests on 11.5.5.1 and everything has passed. Are you ok if I use this version.

---

Also, the version policy for Db2 is >=11.5 which both the oldest and latest satisfy.